docs/users/extend/config_yaml.md

```diff
@@ -26,10 +26,11 @@ the .ddev/config.yaml is the primary configuration for the project.
 | additional_fqdns                 | extra fully-qualified domain names                                                         | `additional_fqdns: ["example.com", "sub1.example.com"]` would provide http and https URLs for "example.com" and "sub1.example.com". Please take care with this because it can cause great confusion and adds extraneous items to your /etc/hosts file.                                                                                                                                                                                         |
 | upload_dir                       | Path from `<docroot>` to the upload directory used as a target by `ddev import-files`      |                                                                                                                                                                                                                                                                                                                                                                                                                                                |
 | working_dir                      | explicitly specify the working directory used by `ddev exec` and `ddev ssh`                | `working_dir: { web:  "/var/www", db: "/etc" }` would set the working directories for the web and db containers.                                                                                                                                                                                                                                                                                                                               |
+| composer_root_dir                | explicitly specify the composer root directory used by `ddev composer`                     | `composer_root_dir: custom-dir` would set the composer root directory to the `custom-dir` sub folder. All composer related commands will run in this directory.                                                                                                                                                                                                                                                                                |
```

Comment:
```suggestion
| composer_root_dir                | explicitly specify the composer root directory used by `ddev composer`                     | `composer_root_dir: /var/www/html/custom-dir` would set the composer root directory to the `custom-dir` sub folder. All composer related commands will run in this directory.                                                                                                                                                                                                                                                                                |
```

Reply:
This is OK (mentioning /var/www/html) but so many people don't understand the difference between in-container and on-host paths...

Reply:
That's currently as the path is always calculated relative to `/var/www/html`. But as said I think we should streamline this configuration and make it absolute too.